game/game.go

```diff
@@ -325,13 +409,36 @@ func (g *Game) PlayMove(m *move.Move, backup bool, addToHistory bool) error {
 		g.scorelessTurns++
 		if addToHistory {
 			turn.Events = append(turn.Events, g.EventFromMove(m))
+			if g.history.LastKnownRacks == nil {
+				g.history.LastKnownRacks = []string{"", ""}
+			}
```

Comment:
This is janky. Why aren't we always setting the LastKnowRacks for every MoveTypeExchange? I guess I don't know why they would even be nil.